language/move-prover/spec-lang/src/ast.rs

```diff
@@ -504,6 +505,13 @@ impl ModuleName {
     pub fn name(&self) -> Symbol {
         self.1
     }
+
+    pub fn is_script(&self) -> bool {
```

Comment:
Is this magic `ff...f` address also used elsewhere? A comment might help explain what's going on here.

Reply:
I found this also a bit confusing so a comment would definitely be helpful. Also, would it be better to possibly use `0x0` instead here possibly? That's the VM reserved address so no account can be created there and ostensibly because of this no module could be published at that address.

Reply:
The magic ff...f comes from the lang framework, I believe the into_module function on scripts creates this. I added a comment to clarify this.